doc/ref/cli/salt-key.rst

```diff
@@ -104,10 +104,20 @@ Actions
 
     Reject the specified public key (use --include-all to match accepted keys
     in addition to pending keys). Globs are supported.
+    .. warning::
+        Using globs to reject the specified public key may cause performance issues
```

Comment:
Is this caused by the `glob` or deleting too many minion's keys at once?

Reply:
it's not related to using globs it happens because of AES key regeneration and then minions mass re-auth. It's enough to delete/reject just one key. See https://docs.saltproject.io/en/latest/topics/tutorials/intro_scale.html#too-many-minions-re-authing

cc: @barbaricyawps